hax/hax/queue/__init__.py

```diff
@@ -82,7 +87,8 @@ def handle_ioq_stob_error(self, payload: Dict[str, Any]) -> None:
         self.queue.put(
             BroadcastHAStates(
                 states=[HAState(fid,
-                                status=ServiceHealth.FAILED)], reply_to=q))
+                                status=ServiceHealth.FAILED)],
+                is_broadcast_local=True, reply_to=q))
         ids: List[MessageId] = q.get()
         self.herald.wait_for_any(HaLinkMessagePromise(ids))
```

Comment:
This is probably one reason why it makes sense to have two separate classes for BroadcastHAStates (one local, another one - cluster-wise).
This "wait_for_any" semantics makes sense if and only if it is local.

But anyway, let's address this later.

Reply:
Okay, so if its a cluster wide broadcast then the ids list will be empty right? do you see any problem here presently?

Reply:
At this moment - no, I don't see issues, that's why I approved it. Just the code can look a bit misleading from the first glance which is not good in longer term.

Reply:
Makes sense. Thanks.